### PR TITLE
Fix CORS schema issues

### PR DESCRIPTION
**Pull request by ivantha**
### Proposed changes in this pull request
$subject.

**References** https://github.com/wso2/product-is/issues/8337

### CODE REVIEW

**Comment by ivantha** [comment]
PR builder passed for this PR. 
https://jenkins-support-all.wso2.com/jenkins/job/DEV_PR_BUILDER/786/

**Review by ashensw — APPROVED** [review]

File: features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/dbscripts/db2.sql [hunk 1]
```diff
@@ -1590,11 +1589,11 @@ CREATE TABLE IDN_USER_FUNCTIONALITY_PROPERTY (
 )
 /
 
-CREATE TABLE IF NOT EXISTS IDN_CORS_ORIGIN (
-    ID                INT           NOT NULL AUTO_INCREMENT,
-    TENANT_ID         INT           NOT NULL,
-    ORIGIN            VARCHAR(2048) NOT NULL,
-    UUID              CHAR(36)      NOT NULL,
+CREATE TABLE IDN_CORS_ORIGIN (
+    ID        INT           NOT NULL AUTO_INCREMENT,
```
Comment (Buddhimah) [inline features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/dbscripts/db2.sql:1593]: Dont think this is a valid Syntax in DB2

Comment (ivantha) [reply features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/dbscripts/db2.sql:1593]: I'll fix it through a different PR.

Comment (ivantha) [reply features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/dbscripts/db2.sql:1593]: https://github.com/wso2/carbon-identity-framework/pull/3126

Comment (ivantha) [reply features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/dbscripts/db2.sql:1593]: Fixed in https://github.com/wso2/carbon-identity-framework/pull/3126 @ashensw

**Pull request merged**